tests/aarch64/checker/README.md

```diff
@@ -0,0 +1,33 @@
+# AArch64 equivalence tester
+
+This directory contains an empirical equivalence tester for AArch64 assembly snippets.
+
+## Usage
+
+So far, the checker can only be used on AArch64 hosts; in principle, however, you can cross-compile and emulate it to
+run on x86_64 system too.
+
+* Copy the kernels to compare into `kernel0.i` and `kernel1.i`.
+* Configure the checker (see below)
+* Build the checker: `make checker`
+* Run the checker: `./checker`
+
+The checker will repeatedly run the assembly snippets with randomly chosen register inputs (GPR and FPR) and compare the final
+register states. The output is the set of registers that have been found to always agree across all tests.
+
+The checker also supports kernels accessing memory buffers, provided the base of the buffers is passed through GPRs. In
+this case, the checker will randomly generate initial memory contents, and compare the final memory contents. When
+memory contents differ, the checker fails.
+
+### Configuration
+
+You can configure the checker by adjusting `harness_config.i`:
+
+* With `#define KERNEL_STACK_SIZE XXX` you can indicate that the kernels assume memory of size `XXX` starting from
+  `sp`. Note that you do not need to specify anything for the kernel(s) to use additional stack below `sp`. However,
+  it is assumed that the kernels restore the original value of `sp` (otherwise, the checker will likely segfault).
+* With `#define FPR_FIX_VALUE_X{0-30} VALUE` you can omit random assignment to a GPR and instead force a given
```

Comment:
Probably a typo: Should be `{G,F}PR_FIX_VALUE_X{0-30}` or `GPR_FIX_VALUE_X{0-30}`, right?

Reply:
No longer relevant after the switch to unicorn.